editor/test/integration/save_data_test.clj

```diff
@@ -603,7 +604,7 @@
   ;; migrations might be covered by tests elsewhere.
   (test-util/with-loaded-project project-path
     (testing "collection"
-      (let [uniform-scale-collection (test-util/resource-node project "/silently_migrated/uniform_scale.collection")
+      (let [uniform-scale-collection (project/get-resource-node project "/silently_migrated/uniform_scale.collection")
```

Comment:
I used to call `test-util/resource-node` and `test-util/resource` instead of the `project` and `workspace` functions in this file. But then I noticed that `test-util/resource` would call `workspace/file-resource`, and not `workspace/find-resource`, as I had expected. This soured me on both functions, so now I call the `project` and `workspace` functions directly.